README.md

```diff
@@ -21,6 +21,7 @@ Spike supports the following RISC-V ISA features:
   - B extension, v0.92
   - K extension, v0.8.1 ([Scalar Cryptography](https://github.com/riscv/riscv-crypto))
   - V extension, v0.9, w/ Zvlsseg/Zvamo/Zvqmac, w/o Zvediv, (_requires a 64-bit host_)
+  - P extension, v0.9.1
```

Comment:
@hope51607 Could you update version info here? I think it should be 0.9.2 according your latest subject of this PR.